<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>521</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>592</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Driver Loader MinGW</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>201</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>201</height>
         </size>
        </property>
        <property name="title">
         <string>Load and Start Driver</string>
        </property>
        <widget class="QLineEdit" name="lineEditDriverPath">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>431</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ButtonBrowseDriver">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>20</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse Driver</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ButtonRegisterDriver">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Register Driver</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ButtonUnregisterDriver">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Unregister Driver</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ButtonStart">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ButtonStop">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>110</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineDriverName">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="maxLength">
          <number>1024</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>53</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>- Driver name</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ButtonDeviceIoControl">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>80</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>DeviceIoControl</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="textEditLog"/>
         </item>
        </layout>
        <zorder>verticalSpacer</zorder>
        <zorder>textEditLog</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
